### PR TITLE
Add tables of analysis commands for every dataset

### PR DESCRIPTION
**Pull request by pgarrison**
# Purpose
Follow-up on @ritvikvasan's [comment](https://github.com/AllenCell/benchmarking_representations/pull/61#discussion_r1859257180) in #61 that we could use the same table format for `run_features.py` and `run_analysis.py`

# Reviewing
Warning: I didn't run these commands myself! These are untested instructions.

@ritvikvasan and @fatwir please read these closely and check they are the same commands you used.

### CODE REVIEW

**Review by vianamp — APPROVED** [review]
LGTM

File: docs/USAGE.md [hunk 1]
```diff
@@ -170,19 +170,27 @@ You can download our pre-computed embeddings here.
...
+| npm1_perturb      | `python src/br/analysis/run_analysis.py --save_path "./outputs_npm1_perturb/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/npm1_perturb" --dataset_name "npm1_perturb" --run_name "Rotation_invariant_pointcloud_jitter_npm1_perturb" --sdf True`                     |
+| npm1              | `python src/br/analysis/run_analysis.py --save_path "./outputs_npm1/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/npm1" --dataset_name "npm1" --run_name "Rotation_invariant_pointcloud_jitter_npm1" --sdf True`                                                     |
+| other_polymorphic | `python src/br/analysis/run_analysis.py --save_path "./outputs_other_polymorphic/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/other_polymorphic" --dataset_name "other_polymorphic" --run_name "Rotation_invariant_pointcloud_jitter_other_polymorphic" --sdf True` |
```
Comment (ritvikvasan) [inline docs/USAGE.md:191]: run name is `Rotation_invariant_pointcloud_SDF`

File: docs/USAGE.md [hunk 2]
```diff
@@ -170,19 +170,27 @@ You can download our pre-computed embeddings here.
...
+| other_punctate    | `python src/br/analysis/run_analysis.py --save_path "./outputs_other_punctate/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/other_punctate" --dataset_name "other_punctate" --run_name "Rotation_invariant_pointcloud_jitter_other_punctate" --sdf False`            |
```
Comment (ritvikvasan) [inline docs/USAGE.md:192]: run_name is `Rotation_invariant_pointcloud_structurenorm`.

File: docs/USAGE.md [hunk 3]
```diff
@@ -170,19 +170,27 @@ You can download our pre-computed embeddings here.
...
-```
+1. To compute benchmarking features from the embeddings and trained models, run the following commands.
+
+| Dataset           | Benchmarking features                                                                                                                                                                                                                                  |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| cellpack          | `python src/br/analysis/run_features.py --save_path "/outputs_cellpack/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/cellpack" --sdf False --dataset_name "cellpack" --debug False`                           |
+| npm1_perturb      | `python src/br/analysis/run_features.py --save_path "/outputs_npm1_perturb/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/npm1_perturb" --sdf True --dataset_name "npm1_perturb" --debug False`                |
```
[the remark this answers is not in the document]
Comment (pgarrison) [reply docs/USAGE.md]: Addressed in [a403119](https://github.com/AllenCell/benchmarking_representations/pull/70/commits/a403119658b2cb6c7a420aa1ba179d51742dc29e)

File: docs/USAGE.md [hunk 4]
```diff
@@ -170,19 +170,27 @@ You can download our pre-computed embeddings here.
...
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| cellpack          | `python src/br/analysis/run_analysis.py --save_path "./outputs_cellpack/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/cellpack" --dataset_name "cellpack" --run_name "Rotation_invariant_pointcloud_jitter_cellpack" --sdf False`                                    |
+| npm1_perturb      | `python src/br/analysis/run_analysis.py --save_path "./outputs_npm1_perturb/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/npm1_perturb" --dataset_name "npm1_perturb" --run_name "Rotation_invariant_pointcloud_jitter_npm1_perturb" --sdf True`                     |
```
[the remark this answers is not in the document]
Comment (pgarrison) [reply docs/USAGE.md]: [a403119](https://github.com/AllenCell/benchmarking_representations/pull/70/commits/a403119658b2cb6c7a420aa1ba179d51742dc29e)

File: docs/USAGE.md [hunk 5]
```diff
@@ -170,19 +170,27 @@ You can download our pre-computed embeddings here.
...
+| other_polymorphic | `python src/br/analysis/run_analysis.py --save_path "./outputs_other_polymorphic/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/other_polymorphic" --dataset_name "other_polymorphic" --run_name "Rotation_invariant_pointcloud_jitter_other_polymorphic" --sdf True` |
+| other_punctate    | `python src/br/analysis/run_analysis.py --save_path "./outputs_other_punctate/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/other_punctate" --dataset_name "other_punctate" --run_name "Rotation_invariant_pointcloud_jitter_other_punctate" --sdf False`            |
+| pcna              | `python src/br/analysis/run_analysis.py --save_path "./outputs_pcna/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/pcna" --dataset_name "pcna" --run_name "Rotation_invariant_pointcloud_jitter_pcna" --sdf False`                                                    |
```
[the remark this answers is not in the document]
Comment (pgarrison) [reply docs/USAGE.md]: Resolved in [ab7f60d](https://github.com/AllenCell/benchmarking_representations/pull/70/commits/ab7f60d508ec8572b78c68ce77cb35c6c5f66b12)

File: docs/USAGE.md [hunk 6]
```diff
@@ -170,19 +170,27 @@ You can download our pre-computed embeddings here.
...
+
+2. To run analysis like latent walks and archetype analysis on the embeddings and trained models, run the following commands.
+
+| Dataset           | Benchmarking features                                                                                                                                                                                                                                                                                         |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| cellpack          | `python src/br/analysis/run_analysis.py --save_path "./outputs_cellpack/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/cellpack" --dataset_name "cellpack" --run_name "Rotation_invariant_pointcloud_jitter_cellpack" --sdf False`                                    |
```
[the remark this answers is not in the document]
Comment (pgarrison) [reply docs/USAGE.md]: [ab7f60d](https://github.com/AllenCell/benchmarking_representations/pull/70/commits/ab7f60d508ec8572b78c68ce77cb35c6c5f66b12)

File: docs/USAGE.md (hunk 1 above)
Comment (pgarrison) [reply docs/USAGE.md]: Resolved in [5b7daed](https://github.com/AllenCell/benchmarking_representations/pull/70/commits/5b7daedaf4eca66999ac25986f2e7da3d3bed009)

File: docs/USAGE.md [hunk 7]
```diff
@@ -170,19 +170,27 @@ You can download our pre-computed embeddings here.
...
+| npm1_perturb      | `python src/br/analysis/run_analysis.py --save_path "./outputs_npm1_perturb/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/npm1_perturb" --dataset_name "npm1_perturb" --run_name "Rotation_invariant_pointcloud_jitter_npm1_perturb" --sdf True`                     |
+| npm1              | `python src/br/analysis/run_analysis.py --save_path "./outputs_npm1/" --embeddings_path "./morphology_appropriate_representation_learning/model_embeddings/npm1" --dataset_name "npm1" --run_name "Rotation_invariant_pointcloud_jitter_npm1" --sdf True`                                                     |
```
[the remark this answers is not in the document]
Comment (pgarrison) [reply docs/USAGE.md]: [5b7daed](https://github.com/AllenCell/benchmarking_representations/pull/70/commits/5b7daedaf4eca66999ac25986f2e7da3d3bed009)

File: docs/USAGE.md (hunk 2 above)
Comment (pgarrison) [reply docs/USAGE.md]: Resolved in [ea50788](https://github.com/AllenCell/benchmarking_representations/pull/70/commits/ea507885bf3247f2b52a30c368a3293def0a9af1)

**Comment by fatwir** [comment]
LGTM!

**Review by ritvikvasan — APPROVED** [review]
Looks good now

**Pull request merged**